scp/cc/azure/attestation/src/utils/hex.cc

```diff
@@ -0,0 +1,41 @@
+/*
+ * Portions Copyright (c) Microsoft Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "hex.h"
+
+namespace google::scp::azure::attestation::utils {
+
+std::vector<uint8_t> decodeHexString(const std::string_view hexstring,
```

Comment:
It might be safer to write test for this or we can use functions like `absl::HexStringToBytes`. absl is already one of the dependencies of the B&A